imagersite/imagerprofile/models.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class ActiveProfileManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveProfileManager, self).get_queryset()\
+            .filter(user__is_active=True)
```

Comment:
There's no reason to use a line continuation character here.  Fix the line length problem in a different way.